spec/person_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './person'
+describe Person do
+  let(:person) { Person.new(31, 'Mumbere Habert') }
+  it 'should initialize correctly' do
+    expect(person.age).to eq(31)
+    expect(person.name).to eq('Mumbere Habert')
+  end
```

Comment:
- kindly make sure constructor method equally returns a person object with empty rentals as @rentals was initialized in the constructor with an empty array as previous reviwer seggested :+1: